scripts/new_project.kts

```diff
@@ -90,35 +124,11 @@ object NewProject {
         }
     }
 
-    private fun cleanNewProjectFolder() {
-        executeCommand("sh $projectPath${File.separator}gradlew -p $projectPath clean")
-        executeCommand("sh $projectPath${File.separator}gradlew -p $projectPath${File.separator}buildSrc clean")
-        listOf(".idea", ".gradle", "buildSrc${File.separator}.gradle", ".git").forEach {
-            File("$projectPath${File.separator}$it")?.let { targetFile ->
-                targetFile.deleteRecursively()
-            }
-        }
-    }
-
     private fun executeCommand(command: String) {
         val process = Runtime.getRuntime().exec(command)
         process.inputStream.reader().forEachLine { println(it) }
     }
```

Comment:
I think we should terminate/kill the process if something fails 💭 

Let's say, we run the Gradle script for running tests and if one of the tests fails, it will still show: `🚀 Done! The project is ready for development`

---

My suggestion is to do something like this:

```
private fun executeCommand(command: String) {
    val process = Runtime.getRuntime().exec(command)
    process.inputStream.reader().forEachLine { println(it) }
    val exitValue = process.waitFor()
    if (exitValue != 0) System.exit(exitValue)
}
```

So if the command resulted in an error, then kill the whole process, otherwise proceed 🤖

Reply:
Shall we only terminate the task or shall we also prompt like `"Something went wrong!!"` before we terminate the process? @Tuubz

Reply:
Yeah, adding an additional error message sounds like a good plan 🚀

Reply:
Can you suggest what we should prompt? @Tuubz 🙏

Reply:
`❌ Something went wrong!` works for m 😁